src/Entity/User.php

```diff
@@ -417,4 +417,14 @@ public function getRoleLabel(): string
         }
         return $role;
     }
+
+    public function getDutiesToString(): string
+    {
+        $dutyCollection = $this->getDuties();
+        $dutyArray = [];
+        foreach ($dutyCollection as $duty) {
```

Comment:
refactorisable, notamment pour simplifier tes noms de variables : 
foreach($this->getDuties as $duty) {
  $dutyNames[] = $duty->getName();
}
return implode(', ', $dutyNames ?? []);

Reply:
Je dois déclarer le tableau dutyNames en dehors de la boucle, sinon PHPmd ne passe pas.